skbio/alignment/_indexing.py

```diff
@@ -180,7 +180,7 @@ def _slice_sequences(self, obj, indexable):
         self._assert_tuple_rules(indexable)
         if (self._has_fancy_index() and
                 type(indexable) is not tuple and
-                pd.core.common.is_list_like(indexable) and
+                pd.api.types.is_list_like(indexable) and
```

Comment:
I made those changes because `pd.core.common.is_list_like` was deprecated, and this warning appeared many times in my tests. I found that the recommended module was `pd.core.common.is_list_like` (in my `pandas` library)